criteria/version-control-and-history.md

```diff
@@ -17,17 +17,18 @@ order: 5
 
 Version control means keeping track of changes to your code over time. This allows you to create structured documentation of the history of the codebase. This is essential for collaboration at scale.
 
-Version control enables you to:
+Version control allows you to:
```

Comment:
This is a definite 'enable'. To allow is to give permission, to enable is to make possible.

Here is an article from Haje on this: https://medium.com/@Haje/in-tech-journalism-you-rarely-allow-you-probably-enable-8eb918a6af15 :)

Reply:
Yup, I agree.